detect/definitions.go

```diff
@@ -134,11 +134,11 @@ func init() {
 	})
 
 	taskList.Add(Task{
-		ID:      taskPhoenixEMCP,
+		ID:      taskPhoenixEth,
 		Type:    "modbus",
 		Depends: TaskTCP502,
 		Config: map[string]interface{}{
-			"ids":     []int{180},
+			"ids":     []int{180, 255},
```

Comment:
Das killt die Wallbe- oder kommt der Wallbe Test noch separat?

Reply:
Ich würde dass gerne mal in einer Session mit dir durchgehen.
Vielleicht sogar überhaupt erst mit der Detect-Routine anfangen und nur davon eine neue Version mergen, unabhängig von den Modulen.
So könnte man auch testen ob meine Vermutungen auch in der Praxis passen ohne im allerschlimmsten Fall dann doch wieder was zurückdrehen zu müssen.